<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stylus BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cadastrar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Cadastrar na Estante</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Livros Concluidos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>140</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Livros em Andamento</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbLidos">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>341</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>8</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>60</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Qtd Páginas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Livro</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="botao_ordenarnome">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ordenar por Ordem Alfabética</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenarnota">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>110</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ordenar por Nota</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>Nome do Livro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnnome"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_37">
         <property name="font">
          <font>
           <family>Stylus BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Autor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnautor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_38">
         <property name="font">
          <font>
           <family>Stylus BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Gênero:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbgenero">
         <item>
          <property name="text">
           <string>Selecione um Gênero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Romance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Comédia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Drama</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ação/Aventura</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Terror</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ficção</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>220</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbstatus">
         <item>
          <property name="text">
           <string>Selecione um status</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Emprestado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Livro em Andamento</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Livro Concluidos</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_14">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>174</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>Nota</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnnota"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_15">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>40</y>
        <width>174</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>Páginas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnrele"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tbanda">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>341</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>8</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>60</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Página</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Livro</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Emprestado?</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbep">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>175</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NÃO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SIM</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Relatórios</string>
     </attribute>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Minha BookShelf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/imagens/375-3758167_twitter-like-icon-png.png</normaloff>../../Desktop/imagens/375-3758167_twitter-like-icon-png.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/imagens/download.jpg</normaloff>../../Desktop/imagens/download.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Números de Livros Lidos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Número de Livros em andamento</string>
      </property>
     </widget>
     <widget class="QLabel" name="nanda">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nlidos">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Total de Livros Emprestados</string>
      </property>
     </widget>
     <widget class="QLabel" name="livrosep">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_57">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total de Livros na Estante</string>
      </property>
     </widget>
     <widget class="QLabel" name="totallivros">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_atualizar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>320</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DADOS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Extra</string>
     </attribute>
     <widget class="QLabel" name="label_59">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sua Estante Online</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_skoob">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/imagens/skoob.png</normaloff>../../Desktop/imagens/skoob.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lelivros">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/imagens/Livro-Logo-SEEL.png</normaloff>../../Desktop/imagens/Livro-Logo-SEEL.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mais Livros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_git">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Desktop/imagens/3-2-github-png-image.png</normaloff>../../Desktop/imagens/3-2-github-png-image.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manual GitHub</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>99</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Skoob</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lelivros.love</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stylus BT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ECarateu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="actionSalvar_Livros_que_Estou_Lendo"/>
    <addaction name="actionCarregar_Para_Ler"/>
    <addaction name="actionSalvarlidos"/>
    <addaction name="actionCarregar_Livros_Lidos"/>
   </widget>
   <addaction name="menuArquivos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar_Livros_que_Estou_Lendo">
   <property name="text">
    <string>Salvar Livros que Estou Lendo</string>
   </property>
  </action>
  <action name="actionCarregar_Para_Ler">
   <property name="text">
    <string>Carregar Para Ler</string>
   </property>
  </action>
  <action name="actionSalvarlidos">
   <property name="text">
    <string>Salvar Livros Lidos</string>
   </property>
  </action>
  <action name="actionCarregar_Livros_Lidos">
   <property name="text">
    <string>Carregar Livros Lidos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
